docs/ops/state/state_backends.zh.md

```diff
@@ -155,7 +155,7 @@ env.setStateBackend(new FsStateBackend("hdfs://namenode:40010/flink/checkpoints"
 {% endhighlight %}
 
 <div class="alert alert-info" markdown="span">
-  <strong>注意:</strong> 由于 RocksDB 是 Flink 默认分发包的一部分，所以如果你没在代码中使用 RocksDB，那就不需要依赖它。而且可以在你的`flink-conf.yaml`文件中通过`state.backend`配置 state backend，以及更多的 [checkpointing]({{ site.baseurl }}/ops/config.html#checkpointing) 和 [RocksDB 特定的]({{ site.baseurl }}/ops/config.html#rocksdb-state-backend) 参数。
+  <strong>注意:</strong> 由于 RocksDB 是 Flink 默认分发包的一部分，所以如果你没在代码中使用 RocksDB，则不需要添加此依赖。而且可以在 `flink-conf.yaml` 文件中通过 `state.backend` 配置 State Backend，以及更多的 [checkpointing]({{ site.baseurl }}/zh/ops/config.html#checkpointing) 和 [RocksDB 特定的]({{ site.baseurl }}/zh/ops/config.html#rocksdb-state-backend) 参数。
```

Comment:
Yes, we should change the PR and issue title.
About the translation. I think flink-conf.yaml as the cluster level and dependency as the job level. If we don't configure RocksDB in configuration file, we don't need to add this dependency. If we configure RocksDB in the configuration file, it will work for the entire cluster by default. RocksDB is part of default distribution without add dependency. So I think the translation maybe right.

Reply:
Hi, @klion26 ,what do you think about this ?